pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -35,13 +35,16 @@ type WatchdogImpl struct{}
 
 // Attest Information Context
 type attestContext struct {
-	zedagentCtx    *zedagentContext
-	attestFsmCtx   *zattest.Context
-	pubAttestNonce pubsub.Publication
+	zedagentCtx               *zedagentContext
+	attestFsmCtx              *zattest.Context
+	pubAttestNonce            pubsub.Publication
+	pubVaultKeyFromController pubsub.Publication
```

Comment:
Why do you name it vault key here and escrow data elsewhere? I think escroweddata makes more sense. Once it has been decrypted it is a vault key

Reply:
Changed it to pubEscrowedDataFromController